apps/gallery/js/ImageEditor.js

```diff
@@ -1517,20 +1583,24 @@ ImageEditor.prototype.cropStart = function(ev) {
       // one dimension will be 100 and will be larger in the other.
       var minWidth = 100, minHeight = 100;
       if (aspectRatio) {
-        if (aspectRatio > 1)
+        if (aspectRatio > 1) {
           minWidth = Math.round(minWidth * aspectRatio);
-        else if (aspectRatio < 1)
+        } else if (aspectRatio < 1) {
           minHeight = Math.round(minHeight / aspectRatio);
+        }
       }
 
       // if the width is less than the minimum allowed (due to orientation
       // change), only allow the crop region to get bigger
       var newWidth = newright - newleft;
-      if ((newWidth < (region.right - region.left)) && (newWidth < minWidth))
+      if ((newWidth < (region.right - region.left)) && (newWidth < minWidth)) {
         return;
+      }
       var newHeight = newbottom - newtop;
-      if ((newHeight < (region.bottom - region.top)) && (newHeight < minHeight))
+      if ((newHeight < (region.bottom - region.top)) &&
+        (newHeight < minHeight)) {
```

Comment:
nit: indent two spaces  to align with the top expression (newHeight < (region.bottom - region.top)) for readability